docs/reference/tab-widgets/inference-api/infer-api-mapping.asciidoc

```diff
@@ -202,6 +202,38 @@ the {infer} pipeline configuration in the next step.
 
 // end::azure-ai-studio[]
 
+// tag::google-vertex-ai[]
+
+[source,console]
+--------------------------------------------------
+PUT google-vertex-ai-embeddings
+{
+  "mappings": {
+    "properties": {
+      "content_embedding": { <1>
+        "type": "dense_vector", <2>
+        "dims": 768, <3>
+        "element_type": "float",
+        "similarity": "dot_product" <4>
+      },
+      "content": { <5>
+        "type": "text" <6>
+      }
+    }
+  }
+}
+--------------------------------------------------
+<1> The name of the field to contain the generated tokens. It must be referenced in the {infer} pipeline configuration in the next step.
```

Comment:
```suggestion
<1> The name of the field to contain the generated embeddings. It must be referenced in the {infer} pipeline configuration in the next step.
```

Should this be `embeddings` instead of `tokens`?